libkbfs/disk_block_cache.go

```diff
@@ -79,6 +82,9 @@ type DiskBlockCacheLocal struct {
 	currBytes uint64
 	// Track the last unref'd revisions for each TLF.
 	tlfLastUnrefs map[tlf.ID]kbfsmd.Revision
+	// Don't evict files from the user's private or public home directory.
+	// Higher numbers are more important not to evict.
+	homeDirs map[tlf.ID]int
```

Comment:
This should probably be initialized in the constructor, just in case someone tries to write to it before they are logged in.